flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/job/ExceptionShowSizeParameter.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.rest.messages.job;
+
+import org.apache.flink.runtime.rest.messages.MessageQueryParameter;
+
+/**
+ * @see org.apache.flink.runtime.rest.handler.job.JobExceptionsHandler
+ * show size for JobExceptionsHandler
+ */
+public class ExceptionShowSizeParameter extends MessageQueryParameter<Integer> {
+
+	private static final String QUERY_PARAMETER_NAME = "size";
+
+	public ExceptionShowSizeParameter() {
+		super(QUERY_PARAMETER_NAME, MessageParameterRequisiteness.OPTIONAL);
+	}
+
+	@Override
+	public Integer convertStringToValue(String value) {
+		return Integer.valueOf(value);
+	}
+
+	@Override
+	public String convertValueToString(Integer value) {
+		return value.toString();
+	}
+
+	@Override
+	public String getDescription() {
+		return "Show exception's size.";
```

Comment:
Parameter descriptions are reasonable standardized. Please have a look at other descriptions and update this one accordingly.

Additionally the description doesn't really make sense; we aren't changing the size of exceptions after all.

Here's a suggestion to save us some time, but it would be good if you would be more diligent in the future:

```suggestion
		return "Integer value that specifies the upper limit of exceptions to return.";
```